packages/components/announcer/src/announcer.ts

```diff
@@ -0,0 +1,79 @@
+import { localized } from '@lit/localize';
+import { ScopedElementsMap } from '@open-wc/scoped-elements/html-element.js';
+import { EventEmitter, EventsController } from '@sl-design-system/shared';
+import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+// import { state } from 'lit/decorators.js';
+import { Announcement } from './announcement.js';
+import styles from './announcer.scss.js';
+
+declare global {
+  interface GlobalEventHandlersEventMap {
+    'sl-announce': SlAnnounceEvent;
+    'sl-announce-polite': SlAnnounceEvent;
+    'sl-announce-assertive': SlAnnounceEvent;
+  }
+
+  interface HTMLElementTagNameMap {
+    'sl-announcer': Announcer;
+  }
+}
+
+export type SlAnnounceEvent = CustomEvent<{ message: string; urgency?: 'polite' | 'assertive' }>;
+
+/** Sends a notification to the live aria.
```

Comment:
Do we have plans to add the component description on the documentation website as well? :)

Reply:
Maybe we need some changes here as well?
https://proud-water-0d02ee203-1649.westeurope.2.azurestaticapps.net/categories/components/progress-bar/accessibility/